src/bolt/utils.ts

```diff
@@ -0,0 +1,93 @@
+import {
+    ReactionAddedEvent,
+    ReactionRemovedEvent
+} from '@slack/bolt';
+import {
+    ReactionData,
+    UserInfo
+} from './types';
+import {
+    ConversationsRepliesResponse
+} from '@slack/web-api/dist/response/ConversationsRepliesResponse';
+import {
+    GenericMessageEvent
+} from '@slack/bolt/dist/types/events/message-events';
+import app from './app';
+
+let botUserId : string | unknown = null;
+
+export async function getBotUserId() {
+    if (!botUserId) {
+        const result = await app.client.auth.test();
+        botUserId = result.user_id;
+    }
+    return botUserId;
+}
+
+// https://api.slack.com/methods/users.profile.get
+export async function getUserInfo(user: string): Promise<UserInfo> {
+    const info = await app.client.users.profile.get({user: user});
+    const {real_name, display_name} = info.profile ?? {};
```

Comment:
Could info be undefined?
```suggestion
    const {profile: { real_name, display_name } = {} } = info?.profile;
```